src/main/java/com/github/bibsysdev/nva/cristin/person/model/nva/NvaIdentifier.java

```diff
@@ -0,0 +1,23 @@
+package com.github.bibsysdev.nva.cristin.person.model.nva;
+
+public class NvaIdentifier {
+
+    public String type;
```

Comment:
We use "type" for indicating the class of the object usually. @brinxmat  should we add type infromation in the json objects?

Reply:
@brinxmat was the one who requested that type be added to the json objects.